libs/client.lua

```diff
@@ -599,6 +599,18 @@ packetListener = {
 		end
 	},
 	[8] = {
+		[1] = function(self, packet, connection, identifier) -- Plays an action
+			if not self._handle_players or self.playerList.count == 0 then return end
+			local playerId = packet:read32()
+			local action = packet:read8()
+			local flags = ""
```

Comment:
`nil` is the correct value to be passed if it doesn't exist
```suggestion
			local flags
```